client/src/components/App.jsx

```diff
@@ -67,24 +72,104 @@ class App extends React.Component {
         }
       )
   }
+  // change state if search is activated
+  changeLocationSearchStatus() {
+    this.setState({
+      locationSearched: true
+    });
+    this.getLocation();
+  }
+
+  getLocation() {
+    console.log('getting location');
+    let latitude;
+    let longitude;
+    navigator.geolocation.getCurrentPosition(
```

Comment:
This is all looking intriguing (and your code is clean and intuitive enough to be understandable within moments of reading it). Is navigator a module you intend to install later on?

Reply:
Navigator is built in(I believe in the browser?). I was not able to use Fandangos API as it is not public so I couldn't simply get data based off zip code etc. I found a different API that requires geolocation coordinates and googled that method to find geolocation for you.